src/cljs/replumb/repl.cljs

```diff
@@ -744,6 +764,7 @@
   supporting:
 
   * :verbose - will enable the the evaluation logging, defaults to false
+  * :filename - the path to a file to load, if any
```

Comment:
Only here, not in the `core`'s `read-eval-call` since this is an implementation detail.